projects/admin/src/app/record/detail-view/library-detail-view/library-detail-view.component.html

```diff
@@ -40,6 +40,13 @@ <h1>{{ record.metadata.name }}</h1>
     <dd class="col-sm-7 col-md-8 mb-0">
       {{ record.metadata.email }}
     </dd>
+    <!-- COMMUNICATION LANGUAGE -->
+    <dt class="col-sm-3 offset-sm-2 offset-md-0 label-title" *ngIf="record.metadata.communication_language" translate>
+      Communication language
+    </dt>
+    <dd class="col-sm-7 col-md-8 mb-0">
+      {{ ('lang_' + record.metadata.communication_language) | translate | ucfirst }}
+    </dd>
```

Comment:
same remarks, especially here where we use "offset-sm" and "offset-md"... coud be change into simpliest `col-3 offset-2` and `col-7`